.github/workflows/ci.yml

```diff
@@ -35,6 +35,6 @@ jobs:
     with:
       gradle-cache-read-only: ${{ github.ref != 'refs/heads/trunk' }}
       gradle-cache-write-only: ${{ github.ref == 'refs/heads/trunk' }}
-      is-public-fork: ${{ github.event.pull_request.head.repo.full_name != 'apache/kafka' }}
+      is-public-fork: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name != 'apache/kafka' }}
```

Comment:
How about using `github.event.pull_request.head.repo.fork == 'true'`? The trunk builds does not have that field.

Reply:
Nice! I didn't notice that field. Much simpler 👍